lib/elasticsearch_models/query/aggregation_term.rb

```diff
@@ -8,12 +8,13 @@ class AggregationTerm
       DEFAULT_ORDER_BY = "_count"
       DEFAULT_ORDER_DIRECTION = "desc"
 
-      def initialize(field:, size: nil, order: nil, partition: nil, num_partitions: nil)
+      def initialize(field:, size: nil, order: nil, partition: nil, num_partitions: nil, missing: nil)
```

Comment:
The changes in this class are missing test coverage. Sorry I didn't point this out when you had questions earlier.

In general, any time a class has changes in functionality there should be corresponding changes/additions to its tests.